src/Symfony/Component/DependencyInjection/Loader/PhpFileLoader.php

```diff
@@ -97,75 +91,54 @@ public function load(mixed $resource, ?string $type = null): mixed
             }
 
             if (\is_object($result) && \is_callable($result)) {
-                $result = $this->callConfigurator($result, new ContainerConfigurator($this->container, $this, $this->instanceof, $path, $resource, $this->env), $path);
```

Comment:
All changes next in this file are reverts of support for dealing with `*Config` objects as return values, while keeping support for returning arrays directly.